.env.example

```
@@ -1,3 +1,11 @@
 TELEGRAM_BOT_TOKEN=your_telegram_bot_token
 GEMINI_API_KEY=your_gemini_api_key
 MONGODB_CONNECTION_STRING=your_mongodb_connection_string
+
+# NextAuth Configuration
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET=jPqrp2IvedEpjTPGPiE7RSd3Wki8NgtYlm7GJ3QpJjYL # Generate with: openssl rand -base64 32
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXTAUTH_SECRET key should go before the NEXTAUTH_URL key

(UnorderedKey)